### PR TITLE
CEDS-1485 Add test for displaying all UCRs on Movements page

### PR DESCRIPTION
**Pull request by MaciejRewera**

### CODE REVIEW

File: app/models/SubmissionPresentation.scala [hunk 1]
```diff
@@ -32,4 +32,12 @@ case class SubmissionPresentation(
 
 object SubmissionPresentation {
   implicit val formats = Json.format[SubmissionPresentation]
+
+  object ActionTypes {
+    val Arrival = "Arrival"
```
Comment (bambuchaAdm) [inline app/models/SubmissionPresentation.scala]: Why strings? Maybe case objects with common `sealed abstract class`

File: test/views/MovementsViewSpec.scala [hunk 2]
```diff
@@ -89,5 +91,30 @@ class MovementsViewSpec extends WordSpec with MustMatchers with Stubs with ViewV
       getElementById(pageWithData, s"dateOfUpdate-$conversationId").text() must be("2019-10-31 00:10")
       getElementById(pageWithData, s"noOfNotifications-$conversationId").text() must be("1")
     }
+
+    "contain MUCR and DUCR if Submission contains both" in {
+      val pageWithData: Html = new movements(mainTemplate)(
+        Seq(
+          (
```
Comment (bambuchaAdm) [inline test/views/MovementsViewSpec.scala]: Is this is for tuple ? Maybe `->` would looks better?

Comment (MaciejRewera) [reply test/views/MovementsViewSpec.scala]: DONE

File: app/models/SubmissionPresentation.scala (hunk 1 above)
Comment (MaciejRewera) [reply app/models/SubmissionPresentation.scala]: Yea, good idea. Changing that.

**Comment by bambuchaAdm** [comment]
:+1: for test on writes

**Pull request merged**